### PR TITLE
gpu profile for dev-server

### PR DESCRIPTION
**Pull request by MSVoss**

### CODE REVIEW

File: etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+# PIConGPU output dependencies ################################################
+#
+spack load pngwriter@0.7.0 %gcc@10.2.0
+
+
+# Environment #################################################################
+#
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
+export PIC_BACKEND="omp2b:native" # running on cpu
```
Comment (psychocoderHPC) [inline etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example]: Should be 

```
export PIC_BACKEND="cuda:70" # running on gpu
```

File: etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+spack load boost@1.78.0 +math +filesystem +system %gcc@10.2.0
+
+# PIConGPU output dependencies ################################################
+#
+spack load pngwriter@0.7.0 %gcc@10.2.0
+
+
+# Environment #################################################################
+#
+#export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BOOST_LIB
```
Comment (psychocoderHPC) [inline etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example]: please remove this line

**Review by psychocoderHPC — CHANGES_REQUESTED** [review]

File: etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example (hunk 1 above)
Comment (MSVoss) [reply etc/picongpu/bash-devServer-hzdr/gpu_picongpu.profile.example]: I completely overlooked that. Now everything should be done.

**Review by psychocoderHPC — APPROVED** [review]

**Comment by psychocoderHPC** [comment]
CI is not providing green because of VM issue. It is not important because the changed scripts will not be checked by the CI.

**Pull request merged**